docs/portal/image-registry/managing-repositories-under-a-project.md

```diff
@@ -0,0 +1,12 @@
+# Managing Repositories Under a Project
+
+A project can have one or more repositories and a repository can have one or more tags. 
+
+The following procedure describes how to view repositories under a project, how to change a repository's description, and how to delete a repository.
+
+1. In the left pane, click **Image Registry > Registry Projects**.
+2. On the Registry Projects page, click the name of a project. The Edit page appears. This page has two sections:<ul><li>**Basic information**  shows read-only information about the selected project.</ul></li><ul><li>**Repositories** shows the repositories associated with the selected project. A field at the top left allows you to filter the repositories shown.</ul></li>
+![null](</docs/resources/images/registry/edit-repository.png>)
+3. To change a repository's description, click the pencil icon in the **Description** column. When the field opens, change the description, and then click the check mark.
+![null](</docs/resources/images/registry/repository-description.png>)
```

Comment:
This screenshot is displayed immediately after the text, instead of being displayed in a separate line. Check the preview, you'll find out. 
This is probably related to the size of a screenshot, and the resolution of laptop screen resolution.

Reply:
Please check now after I applied the width and resolution. It looks fine in Visual Basic. But so did some other items you flagged, so I am not sure that VB always presents an accurate view.

Reply:
The screenshot looks good now. Thanks.